src/main/java/org/mbari/m3/dive/visualizer/DiveService.java

```diff
@@ -10,12 +10,82 @@
 import org.mbari.expd.jdbc.DiveDAOImpl;
 import org.json.simple.JSONObject;
 
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import org.mbari.expd.Dive;
+import org.mbari.expd.DiveDAO;
+import org.mbari.expd.NavigationDatum;
+import org.mbari.expd.NavigationDatumDAO;
+import org.mbari.expd.jdbc.DiveDAOImpl;
+import org.mbari.expd.jdbc.NavigationDatumDAOImpl;
+//import org.mbari.expd.jdbc.NavigationDAOImpl;
+import org.mbari.expd.jdbc.NavigationDatumImpl;
+
+import io.helidon.webserver.Routing;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+import io.helidon.webserver.Service;
+
 public class DiveService implements Service {
 
     @Override // this is called everytime this path is accessed
     public void update(Routing.Rules rules) {
-        rules
-            .get("/{rov}", this::getRovDives);
+        // rules.get("/{rov}", this::getRovDives);
+
+        rules.get("/getlatsandlongs/{rov}/{diveNumber}", (req, res) -> {
```

Comment:
Stick with camel case, same with following methods
```suggestion
        rules.get("/getLatsAndLongs/{rov}/{diveNumber}", (req, res) -> {
```